### PR TITLE
Added chat_join_request to PAYLOAD_TYPES + update API

### PR DESCRIPTION
**Pull request by anko20094**
Thanks for your work!
If I can help you with something - I'll try!

### CODE REVIEW

File: spec/telegram/bot/updates_controller_spec.rb [hunk 1]
```diff
@@ -107,7 +108,7 @@ def action(*args)
 
       context 'when action_missing defined' do
         before do
-          controller.class_eval do
+          controller_class.class_eval do
```
Comment (printercu) [inline spec/telegram/bot/updates_controller_spec.rb:111]: What is this one for?

**Review by printercu — COMMENTED** [review]
Thanks for the PR!
Unfortunately I cannot merge it right now because CI is broken. Travis-ci doesn't work anymore and repo needs to be migrated to GitHub actions.

File: lib/telegram/bot/version.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module Telegram
   module Bot
-    VERSION = '0.15.7'.freeze
+    VERSION = '0.15.8'.freeze
```
Comment (printercu) [inline lib/telegram/bot/version.rb:3]: Please keep it unchanged and add changelog line instead.

**Review by printercu — COMMENTED** [review]
I've set up github workflows. Please rebase onto master branch.

File: spec/telegram/bot/updates_controller_spec.rb (hunk 1 above)
Comment (anko20094) [reply spec/telegram/bot/updates_controller_spec.rb]: Sorry, I've started not correct rspec and had error without that change, but now everything is fine

File: lib/telegram/bot/version.rb (hunk 2 above)
Comment (anko20094) [reply lib/telegram/bot/version.rb]: Done

**Pull request merged**